assets/sass/components/surveys/_googlesitekit-surveys-modal.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * User Feedback styles.
```

Comment:
@asvinb could you please update this header to have the correct information about the content of this file?

Reply:
👍